src/app/app.component.spec.ts

```diff
@@ -13,7 +14,8 @@ describe('AppComponent', () => {
         RouterTestingModule
       ],
       declarations: [
-        AppComponent
```

Comment:
I think you should import the angular modules you used here so you don't fail your CircleCI tests.

Reply:
imported every module now, CI test is OK now, thanks